docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -21,8 +21,8 @@ To complete this guide, you need:
 * less than 15 minutes
 * an IDE
 * JDK 8 installed with `JAVA_HOME` configured appropriately
-* A xref:configuring-c-development[working C development environment]
-* GraalVM version {graalvm-version} installed and xref:configuring-graalvm[configured appropriately]
+* A xref:#configuring-c-development[working C development environment]
+* GraalVM version {graalvm-version} installed and xref:#configuring-graalvm[configured appropriately]
```

Comment:
Hmmm, these links are working correctly on the website. Is it only an issue when you generate locally?

Reply:
This is probably only my personal convention, the # is optional. I'll update the PR.